### DISCUSSION TITLE
Community Page and Posts Support for Tablets

### DISCUSSION THREAD

**Discussion by anil-al** [ReVanced/revanced-discussions#224] [Question]
On tablets, neither Community tab can be used nor posts can be seen on Subscriptions section beside of videos, however it is available for phones and the desktop. Community posts are a big part of channels nowadays and it is a shame that Google haven't implemented it yet so, I'd very like it to be usable by a patch.
![Screenshot_20220614-194425](https://user-images.githubusercontent.com/101948963/173632555-bc0bae34-4f68-44bb-b809-d8109c321b62.png)

**Comment by TheJeterLP** [comment]
This has to get implemented by YouTube.

**Reply by anil-al** [reply]
> This has to get implemented by YouTube.

But it does work in phones so, it isn't really about implementation, rather make the app think it is used in a phone for that spesific tab. Can't that be done?

**Reply by TheJeterLP** [reply]
> > This has to get implemented by YouTube.
> 
> But it does work in phones so, it isn't really about implementation, rather make the app think it is used in a phone for that spesific tab. Can't that be done?

Probably yes, but the UI wont look good. This would be much work, but it is possible tho
